addon/-private/column-tree.js

```diff
@@ -214,6 +220,28 @@ const ColumnTreeNode = EmberObject.extend({
       get(this, 'column.subcolumns').map(column => ColumnTreeNode.create({ column, tree, parent }))
     );
 
+    let widthConstraint = tree.get('widthConstraint');
+
+    if (
```

Comment:
It might be worth pulling this into a CP? What do you think?

```javascript
isSlackModeEnabled: computed('widthConstraint', function() {
  let widthConstraint = tree.get('widthConstraint');
  return widthConstraint === WIDTH_CONSTRAINT.EQ_CONTAINER_SLACK || widthConstraint === WIDTH_CONSTRAINT.GTE_CONTAINER_SLACK;
});
```

Reply:
I like it.